services/github-service-api/src/main/java/org/kontinuity/catapult/service/github/api/GitHubService.java

```diff
@@ -20,8 +26,7 @@
      * @throws NoSuchRepositoryException If the specified repository does not exist
      * @throws IllegalArgumentException  If the repository name is not specified
      */
-    GitHubRepository fork(String repositoryFullName) throws NoSuchRepositoryException,
-            IllegalArgumentException;
+    GitHubRepository fork(String repositoryFullName);
```

Comment:
Curious point you raise, asking the masses. :) https://twitter.com/ALRubinger/status/722932169539067904

Have done it this way for so long never thought of just JavaDocing it.